app/[lang]/layout.tsx

```diff
@@ -20,23 +22,31 @@ export const metadata: Metadata = {
   description: 'Generated by create next app',
 };
 
-export default function RootLayout({
+interface RootLayoutParams {
+  readonly children: ReactNode;
+  readonly params: Promise<{ readonly lang: 'en' | 'uk' }>
```

Comment:
`import { Locale } from "next-intl"`
```suggestion
  readonly params: Promise<{ readonly lang: Locale }>
```